bigchaindb/backend/mongodb/schema.py

```diff
@@ -50,6 +50,10 @@ def drop_database(conn, dbname):
 def create_bigchain_secondary_index(conn, dbname):
     logger.info('Create `bigchain` secondary index.')
 
+    # secondary index on block id which is should be unique
+    conn.conn[dbname]['bigchain'].create_index('id',
```

Comment:
If the index needs to be unique we need to specify it with `unique=True`.

Reply:
Yeah, missed that, my bad.